tests/Unit/PointwiseFunctions/AnalyticData/GrMhd/TestFunctions.py

```diff
@@ -202,3 +202,44 @@ def cylindrical_blast_wave_divergence_cleaning_field(x, inner_radius,
     return 0.0
 
 # End for testing CylindricalBlastWave.cpp
+
+# Functions for testing OrszagTangVortex.cpp
```

Comment:
Better make these separate files?

Reply:
I have no idea how any of this works and just copied from other code.  @osheamonn, I think you're the expert here?

Reply:
I think so far our approach has been to have a single file TestFunctions, which contains the python functions for the given test directory, and if it was decided that a single file was getting too big it could be split. Either way, the file contains bondi hoyle and the cylindrical blast wave functions, which would have to be split out too, so it would be better to do in a different PR, if at all.

Reply:
Since the file is actually separated into sections, I think it would make more sense to split it. We could also get rid of the prefixes for all of the functions then. If you agree I can do that as a separate commit in #1195.

Reply:
#1199